### ISSUE TITLE
Running chmod locally in task

### ISSUE THREAD

**Issue by marcovtwout** [capistrano/sshkit#375]
After downloading a file in a task using:

`download! remote_file local_file`

How can I chmod local_file?

**Comment by marcovtwout** [comment]
Sorry, I missed that topic. When you click New issue, this link goes to another contributing document that does not mension questions:
![image](https://cloud.githubusercontent.com/assets/438046/19651931/2c2959a6-9a0f-11e6-9b6f-15febb8bca80.png)
(points to https://github.com/capistrano/capistrano/blob/master/CONTRIBUTING.md)


Thanks for the help anyways. :)

**Comment by mattbrictson** [comment]
Good catch; we should update the CONTRIBUTING doc for this repo. Thanks!
